### ISSUE TITLE
[1.10 Bug] Unable to turn off tool models

### ISSUE THREAD

**Issue by MechWarrior99** [SonarSonic/Calculator#158]
Switching tool models off does not work.

**Comment by SonarSonic** [comment]
Yeah that was a 1.7.10 feature I haven't removed from config, I won't be adding it back since you can do that with resource packs anyway.

**Comment by MechWarrior99** [comment]
Wont add the option to turn them off? If this is true, would you reconsider if I gave you some nice tool textures so that others who also don't like the modeled tools don't have to go get a resource pack?

**Comment by SonarSonic** [comment]
I won't be adding the .jsons for normal tools.

**Comment by MechWarrior99** [comment]
Okay

**Comment by Fortist** [comment]
Does this still hold true for 1.12?
Because the models still suck.

Maybe even remove the option in the config?

**Comment by SonarSonic** [comment]
*still awesome
I will remove from config in next update.
